### ISSUE TITLE
Create "Lighthouse" concepts article

### ISSUE THREAD

**Issue by StackScribe** [keptn/keptn.github.io#1207]
Add a "Concepts" article about lighthouse -- or incorporate this into the "Intro to Keptn" doc.

The README.md for lighthouse provides good source.

Adam Gartner also provided some useful text in response to an inquiry in the keptn help channel:
[Jurgen Weber](https://app.slack.com/team/U036E4VL4MR) [Apr 19th at 10:40 PM](https://keptn.slack.com/archives/CNRCGFU3U/p1650433242624579)
trying to run through this tutorial; https://tutorials.keptn.sh/tutorials/keptn-full-tour-prometheus-011/index.html?index=..%2F..index#9
The carts-db deployment worked fine, dev/staging/prod.. but now its trying to deploy carts and it seems to be stuck. I can find no secondary signs that hint at an issue.. except that it has been stuck at this stage for nearly an hour:
image.png 
image.png

9 replies

[Adam Gardner](https://app.slack.com/team/UGXUEL7PD)  [5 days ago](https://keptn.slack.com/archives/CNRCGFU3U/p1650433676327819?thread_ts=1650433242.624579&cid=CNRCGFU3U)
Hi Jurgen. The way that works is the lighthouse-service responds to evaluation.triggered and sends an evaluation.started (you can see this event in your screenshot). It also sends a get-sli.triggered automatically (within the code of the lighthouse-service). Then some service (in your case the prometheus-service is supposed to respond to the get-sli.triggered event with get-sli.started and get-sli.finished  - finally the lighthouse-service is also configured to listen for get-sli.finished so it knows it can proceed and judge the evaluation.
In other words, the lifecycle is always .triggered  then .started then .finished
It looks to me like you either haven't got a prometheus-service or it isn't configured to listen for the get-sli.triggered event. I can see that because your screenshot shows no get-sli.started event (underneath the get-sli "box"

**Comment by DavidPHirsch** [comment]
outdated
